src/k8s/cmd/k8s/k8s_x_wait_for_dns.go

```diff
@@ -0,0 +1,20 @@
+package k8s
+
+import (
+	cmdutil "github.com/canonical/k8s/cmd/util"
+	"github.com/canonical/k8s/pkg/k8sd/features"
+	"github.com/spf13/cobra"
+)
+
+func newXWaitForDNSCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
+	waitForDNSCmd := &cobra.Command{
+		Use:    "x-wait-for-dns",
```

Comment:
Are we set on the name of this command? I would like to suggest something like feature-ready or feature-status?

Reply:
I prefer the current naming. `dns-ready` or `dns-status` sound like querying a state while `wait-for-dns` clearly shows that this is a blocking invocation.

Reply:
I agree with Ben. This naming convention is more self-descriptive of the command's purpose.